scripts/db/mongo-dev-init/MOCK_USERS.json

```diff
@@ -0,0 +1,1719 @@
+[
+  {
+    "firstName": "Testy",
+    "lastName": "McTesterson",
+    "email": "tester@codehammers.com",
+    "profilePic": "https://www.codesmith.io/hubfs/Screen%20Shot%202024-06-10%20at%2010.46.24%20AM.png",
+    "password": "$2a$10$LGAtDH0sMsL39sZxWRN8r.X.//5/XBIirtADsSg9VLUciCxXhmtF.",
+    "_id": {
+      "$oid": "666cbc4240af7b375e352e8c"
+    },
+    "lastVisit": {
+      "$date": "2024-06-14T21:55:14.814Z"
+    },
+    "date": {
+      "$date": "2024-06-14T21:55:14.814Z"
+    },
+    "__v": 0
+  },
```

Comment:
Hey @seantokuzo  - Nice to see the site has so many active users! While Testy McTesterson seems like a huge asset to the team, can we add a few teammates for them? Specifically, let's add Jane@codehammers.com, john@codehammers.com, and jaime@ codehammers.com. They're all great friends so don't mind sharing the same "ilovetesting" password.